src/services/sns.ts

```diff
@@ -0,0 +1,26 @@
+import { SNSClient, PublishCommand } from '@aws-sdk/client-sns';
+import logger from '../util/logger';
+
+const snsClient = new SNSClient({ region: process.env.DYNAMO_AWS_REGION });
+
+export const publish = async (message: string, topicArn: string) => {
+  logger.info('Sending SNS notification');
+
+  if (process.env.AWS_SAM_LOCAL) {
+    return '123';
+  }
+  const input = {
+    TopicArn: topicArn,
+    Message: message,
+  };
+
+  const command = new PublishCommand(input);
+  try {
+    const response = await snsClient.send(command);
+    logger.debug(response);
+    return undefined;
+  } catch (err: unknown) {
+    logger.error(err);
+    throw new Error(err as string);
```

Comment:
any opinions on whether the SNS failure should throw an error or silently fail?

Reply:
I feel like an error probably makes sense, at least then there is some kind of trail of whats happened